src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -1490,6 +1550,31 @@ public IEnumerable<string> GetCountersForDocument(DocumentsOperationContext cont
                 yield return c;
         }
 
+        public long GetNumberOfCountersAndDeletedCountersForDocument(DocumentsOperationContext context, string docId)
```

Comment:
add a comment that this is for testing purposes only, maybe mark it as internal

Reply:
done